### PR TITLE
Connection: release a new version of the package and dependents

### PR DESCRIPTION
**Pull request by jeherve**
#### Changes proposed in this Pull Request:

Release a new version of the package to be able to ship #19439 with 9.6.1.

**Note** This also includes a yarn.lock update, but this should be fixed with #19456 

#### Jetpack product discussion

* N/A

#### Does this pull request change what data or activity we track or use?

* No

#### Testing instructions:

* Check for typos

### CODE REVIEW

File: projects/packages/a8c-mc-stats/CHANGELOG.md [hunk 1]
```diff
@@ -5,6 +5,8 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [1.4.3] - 2021-04-08
```
Comment (anomiex) [inline projects/packages/a8c-mc-stats/CHANGELOG.md:8]:
```suggestion
## [1.4.3] - 2021-04-08
### Changed
- Packaging and build changes, no change to the package itself.
```

File: projects/packages/assets/CHANGELOG.md [hunk 2]
```diff
@@ -5,6 +5,8 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [1.11.4] - 2021-04-08
```
Comment (anomiex) [inline projects/packages/assets/CHANGELOG.md:8]:
```suggestion
## [1.11.4] - 2021-04-08
### Changed
- Packaging and build changes, no change to the package itself.
```

File: projects/packages/constants/CHANGELOG.md [hunk 3]
```diff
@@ -5,6 +5,8 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [1.6.4] - 2021-04-08
```
Comment (anomiex) [inline projects/packages/constants/CHANGELOG.md:8]:
```suggestion
## [1.6.4] - 2021-04-08
### Changed
- Packaging and build changes, no change to the package itself.
```

File: projects/packages/heartbeat/CHANGELOG.md [hunk 4]
```diff
@@ -5,6 +5,8 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [1.3.5] - 2021-04-08
```
Comment (anomiex) [inline projects/packages/heartbeat/CHANGELOG.md:8]:
```suggestion
## [1.3.5] - 2021-04-08
### Changed
- Packaging and build changes, no change to the package itself.
```

File: projects/packages/options/CHANGELOG.md [hunk 5]
```diff
@@ -5,6 +5,8 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [1.11.4] - 2021-04-08
```
Comment (anomiex) [inline projects/packages/options/CHANGELOG.md:8]:
```suggestion
## [1.11.4] - 2021-04-08
### Changed
- Packaging and build changes, no change to the package itself.
```

File: projects/packages/roles/CHANGELOG.md [hunk 6]
```diff
@@ -5,6 +5,8 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [1.4.4] - 2021-04-08
```
Comment (anomiex) [inline projects/packages/roles/CHANGELOG.md:8]:
```suggestion
## [1.4.4] - 2021-04-08
### Changed
- Packaging and build changes, no change to the package itself.
```

File: projects/packages/status/CHANGELOG.md [hunk 7]
```diff
@@ -5,6 +5,8 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [1.7.4] - 2021-04-08
```
Comment (anomiex) [inline projects/packages/status/CHANGELOG.md:8]:
```suggestion
## [1.7.4] - 2021-04-08
### Changed
- Packaging and build changes, no change to the package itself.
```

File: projects/packages/terms-of-service/CHANGELOG.md [hunk 8]
```diff
@@ -5,6 +5,8 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [1.9.5] - 2021-04-08
```
Comment (anomiex) [inline projects/packages/terms-of-service/CHANGELOG.md:8]:
```suggestion
## [1.9.5] - 2021-04-08
### Changed
- Packaging and build changes, no change to the package itself.
```

File: projects/packages/tracking/CHANGELOG.md [hunk 9]
```diff
@@ -5,6 +5,8 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [1.13.4] - 2021-04-08
```
Comment (anomiex) [inline projects/packages/tracking/CHANGELOG.md:8]:
```suggestion
## [1.13.4] - 2021-04-08
### Changed
- Packaging and build changes, no change to the package itself.
```

**Review by anomiex — COMMENTED** [review]
Heh, lots of packages where the only change was enabling the auto-tagger.

**Review by anomiex — APPROVED** [review]

**Comment by jeherve** [comment]
Cherry-picked to `jetpack/branch-9.6` in 769eaa1b69f7e58387a310da334aff4fa66c94c5

**Pull request merged**